src/EditorFeatures/CSharpTest/Diagnostics/Async/AddAwaitTests.cs

```diff
@@ -818,7 +874,7 @@ class TestClass
     private async Task MyTestMethod1Async()
     {
         Action @delegate = delegate {
-            int myInt = MyInt [||] MethodAsync();
+            int myInt = {|CS4034:await MyIntMethodAsync()|};
```

Comment:
So if the code fix generates a new diagnostic, which is not handled by the fixer, do we _have to_ annotated the fixed code with the new diagnostic OR _we can_ annotate the fixed code with the new diagnostic if we want to validate it?

Reply:
Everything considered error severity with `/warnAsError:Nullable` set must be declared.

Reply:
Sounds good.